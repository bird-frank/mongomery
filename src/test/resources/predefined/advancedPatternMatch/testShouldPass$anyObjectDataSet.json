{
  "Vocabulary": [
    {
      "_id": {
        "$oid": "55f3ed00b1375a48e61830bf"
      },
      "complexObj": {
        "f1": "v1",
        "f2": "v2",
        "f3": "v3"
      }
    }
  ]
}