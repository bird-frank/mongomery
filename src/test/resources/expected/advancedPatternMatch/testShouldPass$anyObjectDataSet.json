{
  "Vocabulary": [
    {
      "_id": {
        "$oid": "55f3ed00b1375a48e61830bf"
      },
      "complexObj": "$anyObject(3)"
    }
  ]
}